CMakeLists.txt

```diff
@@ -331,21 +331,23 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE
 ###############################################################################
 # Test
 ###############################################################################
-option(BUILD_TESTS "Build Fast DDS Monitor library and documentation tests" OFF)
+option(BUILD_TESTS "Build Fast DDS Monitor appliction and documentation tests" OFF)
+option(BUILD_APP_TESTS "Build Fast DDS Monitor application tests" OFF)
 option(BUILD_DOCUMENTATION_TESTS "Build Fast DDS Monitor documentation tests" OFF)
 
 if (BUILD_TESTS)
+    set(BUILD_APP_TESTS ON)
     set(BUILD_DOCUMENTATION_TESTS ON)
```

Comment:
NIT: This statement is repeated in `docs/CMakeLists.txt` in line 52.

Reply:
Yes, that's because you can compile the documentation without compiling the monitor.